source/slang/slang-ir-autodiff-unzip.h

```diff
@@ -262,19 +280,41 @@ struct DiffUnzipPass
 
     IRBlock* getUpdateBlock(IndexedRegion* region)
     {
+        // TODO: What if the 'continue' region has multiple
```

Comment:
Add a release assert here to make sure the continue block ends with a unconditional branch back to loop head. If this is not the case we should just report failure/crash, so it is easy to fix in the future.